src/Illuminate/Console/ConfirmableTrait.php

```diff
@@ -15,6 +15,10 @@
      */
     public function confirmToProceed($warning = 'Application In Production!', $callback = null)
     {
+        if (! $this->input->isInteractive()) {
+            return true;
```

Comment:
That's kind of the point of the PR. when you run in non-interactive, I'm assuming you know you want to run that command. It's not an 'accident' or something. So when you put `php artisan migrate --non-interactive` in there, you want to run it always right? (So same as --force)